pkg/storage/azureblob.go

```diff
@@ -87,3 +87,15 @@ func UploadFileToSasUrl(blobSasUrl url.URL, content []byte) error {
 	}
 	return nil
 }
+
+func UploadConnectorLog(connectorLoggingUrl string, logs []byte) (string, error) {
```

Comment:
may be good to remove the return `string` since its passed empty always

Reply:
`err = UploadFileToSasUrl(*parse, logs)`
In the implementation part, I'm checking on the return type `error`, if it is not `nil` then it's a happy flow, otherwise it returns the error.

Reply:
Removed string return type. Thanks for the feedback.